src/tests/gssapi/t_ccselect.py

```diff
@@ -114,9 +117,12 @@
 if output != (zaphod + '\n'):
     fail('zaphod not chosen via .k5identity service/host line.')
 output = r1.run(['./t_ccselect', refserver])
-if output != (bob + '\n'):
-    fail('bob not chosen via primary cache when no .k5identity line matches.')
+if output != (alice + '\n'):
+    fail('alice not chosen via primary cache when no .k5identity line matches.')
 r1.run(['./t_ccselect', 'h:bogus@' + hostname], expected_code=1,
-       expected_msg="Can't find client principal noprinc")
+       expected_msg=('gss_init_sec_context: Unspecified GSS failure.  '
+                     'Minor code may provide more information\n'
+                     'gss_init_sec_context: Server bogus/localhost@KRBTEST.COM '
```

Comment:
The local hostname is included in this expected message, which will cause tests to fail whenever the canonicalized local hostname is not "localhost".